packages/addons/addon-depends/vdr-plugins/vdr-plugin-live/patches/vdr-plugin-live-01-makefile.patch

```diff
@@ -0,0 +1,35 @@
+diff --git a/Makefile b/Makefile
+--- a/Makefile
++++ b/Makefile
+@@ -11,7 +11,7 @@ PLUGIN := live
+ VERSION := $(shell grep '\#define LIVEVERSION ' setup.h | awk '{ print $$3 }' | sed -e 's/[";]//g')
+ 
+ ### Check for libpcre c++ wrapper
+-HAVE_LIBPCRECPP := $(shell pcre-config --libs-cpp)
++HAVE_LIBPCRECPP := $(shell pcre-config --libs libpcrecpp libpcre)
+ 
+ ### The directory environment:
+ # Use package data if installed...otherwise assume we're under the VDR source directory:
+@@ -35,17 +35,17 @@ APIVERSION := $(call PKGCFG,apiversion)
+ -include $(PLGCFG)
+ 
+ ### Determine tntnet and cxxtools versions:
+-TNTVERSION = $(shell tntnet-config --version | sed -e's/\.//g' | sed -e's/pre.*//g' | awk '/^..$$/ { print $$1."000"} /^...$$/ { print $$1."00"} /^....$$/ { print $$1."0" } /^.....$$/ { print $$1 }')
+-CXXTOOLVER = $(shell cxxtools-config --version | sed -e's/\.//g' | sed -e's/pre.*//g' | awk '/^..$$/ { print $$1."000"} /^...$$/ { print $$1."00"} /^....$$/ { print $$1."0" } /^.....$$/ { print $$1 }')
++TNTVERSION = $(shell pkg-config --modversion tntnet | sed -e's/\.//g' | sed -e's/pre.*//g' | awk '/^..$$/ { print $$1."000"} /^...$$/ { print $$1."00"} /^....$$/ { print $$1."0" } /^.....$$/ { print $$1 }')
++CXXTOOLVER = $(shell pkg-config --modversion cxxtools | sed -e's/\.//g' | sed -e's/pre.*//g' | awk '/^..$$/ { print $$1."000"} /^...$$/ { print $$1."00"} /^....$$/ { print $$1."0" } /^.....$$/ { print $$1 }')
+ 
+-CXXFLAGS  += $(shell tntnet-config --cxxflags)
+-LIBS      += $(shell tntnet-config --libs)
++CXXFLAGS  += $(shell pkg-config --cflags tntnet)
++LIBS      += $(shell pkg-config --libs tntnet)
+ 
+ ### Optional configuration features
+ PLUGINFEATURES :=
+ ifneq ($(HAVE_LIBPCRECPP),)
+ 	PLUGINFEATURES += -DHAVE_LIBPCRECPP
+-	CXXFLAGS       += $(shell pcre-config --cflags)
++	CXXFLAGS       += $(shell pkg-config --cflags libpcreposix libpcre)
+ 	LIBS           += $(HAVE_LIBPCRECPP)
```

Comment:
Enabling live plugin results in:

```Dec 20 22:42:29 LibreELEC vdr[10346]: [10346] ERROR: /storage/.kodi/addons/service.multimedia.vdr-addon/plugin/libvdr-live.so.2.3.8: undefined symbol: _ZN7pcrecpp2RE6no_argE```

Using `LIBS           += -Wl,-Bstatic $(HAVE_LIBPCRECPP) -Wl,-Bdynamic` solves it but hopefully someone knows a less hackish solution.